includes/include-dir.inc.php

```diff
@@ -5,16 +5,20 @@
     $include_dir_regexp = '/\.inc\.php$/';
 }
 
+$tmp_files = array();
 if ($handle = opendir($config['install_dir'].'/'.$include_dir)) {
```

Comment:
scandir() and glob() both return sorted arrays of files.

`glob($config['install_dir'].'/'.$include_dir.'/*.inc.php');` for example.